src/styles/index.scss

```diff
@@ -1,3 +1,169 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  border: none;
```

Comment:
Don't set margins/paddings/border to all elements, reset only for the elements that you actually need

Reply:
it is just standard simple reset , why not ?

Reply:
<img width="1083" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/86789161/bf8c95c0-79c8-442d-bccb-250367341e41">

Reply:
> it is just standard simple reset , why not ?

It's not, a lot of tags don't have any margins/paddings so you don't need to set them